src/custom/state/index.ts

```diff
@@ -32,12 +31,11 @@ const UNISWAP_REDUCERS = {
 const reducers = {
   ...UNISWAP_REDUCERS,
   lists,
-  operator,
   orders,
   fee
 }
 
-const PERSISTED_KEYS: string[] = ['user', 'transactions', 'orders', 'fee', 'lists']
```

Comment:
imo persisting `fee` in localstorage here is no bueno. Why? because now it is recalculated/requested often from the backend and adding save option to localstorage seems excessive and probably lower performance seeing as how often this may need to happen.

Additionally, user's coming from the old fee state would cause problems in FE unless we implment some sort of schema on the fee state for checking against the `load` state from `localStorage` as it is done before any actions are fired. not as easy as it sounds unfortuately